samples/emoji-search/launcher/src/commonMain/kotlin/com/example/redwood/emojisearch/launcher/EmojiSearchAppSpec.kt

```diff
@@ -19,13 +19,15 @@ import app.cash.redwood.treehouse.TreehouseApp
 import app.cash.zipline.Zipline
 import com.example.redwood.emojisearch.treehouse.EmojiSearchPresenter
 import com.example.redwood.emojisearch.treehouse.HostApi
+import com.example.redwood.emojisearch.treehouse.emojiSearchSerializersModule
 import kotlinx.coroutines.flow.Flow
 
 class EmojiSearchAppSpec(
   override val manifestUrl: Flow<String>,
   private val hostApi: HostApi,
 ) : TreehouseApp.Spec<EmojiSearchPresenter>() {
   override val name = "emoji-search"
+  override val serializersModule = emojiSearchSerializersModule
```

Comment:
It took a while to notice that the `serializersModule` had to be set in _two_ places. I remembered that `Zipline.get()` took a `serializersModule`, but didn't realise that the `TreehouseApp.Spec` also needed one. The error was very unhelpful, as it just said that there was a `serializer` missing for some type.

The stacktrace was very unhelpful, as it just said that there are no serializers module attached for some type. This led to me fiddling with how the serializers are set up, without realising that I needed to attach _another_ `serializersModule` somewhere. I eventually figured it out by doing a global search of `EmptySerializersModule` and found out that I can override the `serializersModule` of `TreehouseApp.Spec`. So I ultimately figured it out by happenstance. 

Is there a way to merge the two of them, so we only need one? (In the case of Emoji Search, it needed to be set in _three_ places!)

Reply:
I think the best thing is to define a serializersModule instance somewhere in your app’s commonMain, then to use that shared instance everywhere Treehouse is initialized.

We can’t really share the initialization code as it’s not in commonMain.

Reply:
> I think the best thing is to define a serializersModule instance somewhere in your app’s commonMain, then to use that shared instance everywhere Treehouse is initialized.

Yep, that's what I did with `emojiSearchSerializersModule`. The `serializersModule` is defined once and referenced where needed.

> We can’t really share the initialization code as it’s not in commonMain.

👍